defaults.gpr

```diff
@@ -58,7 +58,7 @@ abstract project Defaults is
 
    Proof_Switches :=
    (
-      "--prover=z3,cvc4,altergo",
+      "--prover=z3,cvc4,colibri",
```

Comment:
This is just for this particular code base. In general Alt-Ergo and COLIBRI will prove different things. We could move Alt-Ergo last so that it doesn't cost any extra time in the case where everything is proved, but it is still run in the case of unproved VCs.

Reply:
I needed to add alt-ergo back by the way.